docs/database_migrations.md

```diff
@@ -0,0 +1,49 @@
+# Database migrations
+For most of the releases database schema differs between versions. To easily migrate/deploy database schema we are using EntityFramework migrations.
+
+For further reading, visit official Microsoft documentation on entity framework migration https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli
+
+## Executing migrations
+There are two supported methods of applying migrations. Executing Idempotent SQL script or using built docker image. You can choose whichever you prefer.
```

Comment:
This section is slightly misleading. You can either execute the idempotent SQL script or run "programmatic EFCore migrations" (.NET application). The latter is additionally available as a docker image.